models/issue_comment.go

```diff
@@ -235,6 +236,22 @@ func (c *Comment) HTMLURL() string {
 	return fmt.Sprintf("%s#%s", c.Issue.HTMLURL(), c.HashTag())
 }
 
+// APIURL formats a API-string to the issue-comment
+func (c *Comment) APIURL() string {
+	err := c.LoadIssue()
+	if err != nil { // Silently dropping errors :unamused:
+		log.Error("LoadIssue(%d): %v", c.IssueID, err)
+		return ""
+	}
+	err = c.Issue.loadRepo(x)
+	if err != nil { // Silently dropping errors :unamused:
+		log.Error("loadRepo(%d): %v", c.Issue.RepoID, err)
+		return ""
+	}
+
+	return c.Issue.Repo.APIURL() + "/" + path.Join("issues/comments", fmt.Sprint(c.ID))
```

Comment:
Probably better to just use fmt.Sprintf for the whole string:

```suggestion
	return fmt.Sprintf("%s/issues/comments/%d", c.Issue.Repo.APIURL(), c.ID)
```

(In general strconv.Itoa should also be used in preference to fmt.Sprint(...) for stringing integers.)

Reply:
Issue use this too (**strconv.Itoa** cant process **int64**) 
https://github.com/go-gitea/gitea/blob/480ef9d410a5c94c0ce4d16b922be3b1114d620c/models/issue.go#L327

Reply:
I changed it to use fmt.Sprintf ...